### PR TITLE
Updated README to point to correct stub examples

### PR DESCRIPTION
**Pull request by srm09**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ the second argument will be used as the body of the command. (code that would be
 Once you're done, you can delete it with `unstub_command`.
 
 The best example is the [shpec test for this feature](
-https://github.com/rylnd/shpec/blob/master/shpec/shpec_shpec.sh#L72-L89).
+https://github.com/rylnd/shpec/blob/master/shpec/shpec_shpec.sh#L121-L139).
 <!-- beware: keep in sync of line when modifying the shpec -->
```
Comment (rylnd) [inline README.md]: further proof that comments are useless!

**Comment by rylnd** [comment]
@srm09 thanks for this! I'll merge once I figure out the CI issues.

File: README.md (hunk 1 above)
Comment (AdrieanKhisbe) [reply README.md]: ^^

Comment (rylnd) [reply README.md]: Haha,  sorry @AdrieanKhisbe ! We tried.

Comment (AdrieanKhisbe) [reply README.md]: Don't be :D

**Pull request merged**